plugin/api.go

```diff
@@ -179,6 +179,9 @@ type API interface {
 	// UpdatePost updates a post.
 	UpdatePost(post *model.Post) (*model.Post, *model.AppError)
 
+	// GetEmoji returns a custom emoji based on the emojiId string.
```

Comment:
Can you document the minimum version required here?

```suggestion
	// GetEmoji returns a custom emoji based on the emojiId string.
	//
	// Minimum server version: 5.6
```

Reply:
@jwilander Are we documenting minimum server version for these? I don't know how plugins handle different server versions at this point

Reply:
I just saw that a minimum version has been added for other APIs as well, so we should add it here as well

Reply:
We are, @hanzei is leading the effort :)